relayer/main/main.go

```diff
@@ -170,10 +171,14 @@ func main() {
 		})
 	}
 
+	trackedSubnets := set.NewSet[ids.ID](len(cfg.SourceBlockchains))
+	for _, sourceChain := range cfg.SourceBlockchains {
```

Comment:
Do we need to consider the case where a non-converted Subnet is still using the self-signing optimization for messages from the C-Chain here? If for whatever reason that chain is only listed as a destination, we probably still want to add it as a tracked subnet since we will attempt to aggregate signatures from it.

Reply:
Indeed, thanks for the catch